CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [FEATURE] AlertManager: Add support for SNS Receiver. #4382
 * [FEATURE] Distributor: Add label `status` to metric `cortex_distributor_ingester_append_failures_total` #4442
 * [FEATURE] Queries: Added `present_over_time` PromQL function, also some TSDB optimisations. #4505
+* [FEATURE] Ruler: Add `external_labels` option to tag all alerts with a given set of labels.
```

Comment:
Is it only alerts, or recording rules too?

Reply:
As far as I can tell, this only applies to alerts. It gets passed to the notifier configuration specifically, and I don't see any mention of external labels in [vendor/github.com/prometheus/prometheus/rules/recording.go](https://github.com/cortexproject/cortex/blob/master/vendor/github.com/prometheus/prometheus/rules/recording.go).

The [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) also makes it clear that they apply when communicating with external systems:

> The labels to add to any time series or alerts when communicating with external systems (federation, remote storage, Alertmanager).

Reply:
Can you move the changelog to `## master / unreleased`

Reply:
Done.